src/Tizen.NUI/src/public/BaseComponents/ViewAccessibilityEvent.cs

```diff
@@ -64,13 +74,47 @@ public override bool Equals(System.Object obj)
               eventTime == sec.eventTime;
         }
 
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public static bool operator ==(GestureInfoType arg1, GestureInfoType arg2)
+        {
+            return arg1.Equals(arg2);
+        }
+
+        /// <summary>
+        /// The != operator.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public static bool operator !=(GestureInfoType arg1, GestureInfoType arg2)
+        {
+            return !arg1.Equals(arg2);
+        }
+
         [EditorBrowsable(EditorBrowsableState.Never)]
         public override int GetHashCode()
         {
             return Tuple.Create((int)type, xBeg, xEnd, yBeg, yEnd, (int)state, eventTime).GetHashCode();
         }
     }
 
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public class GestureInfoEventArgs : EventArgs
+    {
+        public GestureInfoType GestureInfo { get; internal set; }
+        public int Consumed { get; set; }
```

Comment:
I suppose we should have "consumed by at least one" semantics here, to avoid impact of EventHandlers registration order on the result. What about overriding the set method for Consumed to aggregate current value (projected to {0,1}) with the previous value of this property with OR.